scipy/spatial/transform/_rotation_groups.py

```diff
@@ -0,0 +1,195 @@
+            Object containing the elements of the rotation group.
+        """
+        g1 = self.tetrahedral().as_rotvec()
+        g2 = np.pi / np.sqrt(2) * np.array([[1, 1, 0], [1, 0, 1],
+                                            [0, 1, 1], [1, -1, 0],
+                                            [1, 0, -1], [0, 1, -1]])
+        g3 = np.pi / 2 * np.eye(3)
+        return Rotation.from_rotvec(np.concatenate((g1, g2, g3, -g3)))
+
+    @classmethod
+    def tetrahedral(self):
+        """Get the tetrahedral rotation group.
+
+        Returns
+        -------
+        rotation : `Rotation` instance of length 12
+            Object containing the elements of the rotation group.
+        """
+        g1 = np.concatenate(([np.zeros(3)], np.pi * np.eye(3)))
+        g2 = np.array(list(itertools.product([-1., 1.], repeat=3)))
+        g2 *= 2 * np.pi / (3 * np.sqrt(3))
+        return Rotation.from_rotvec(np.concatenate((g1, g2)))
+
+    @classmethod
+    def dicyclic(self, n, axis=2):
+        """Get a dicyclic rotation group.
+
+        Parameters
+        ----------
+        n : integer
+            The order of the group.
+        axis: integer
+            The cyclic rotation axis (default: 2)
+            (0: x-axis, 1: y-axis, 2: z-axis)
+
+        Returns
+        -------
+        rotation : `Rotation` instance of length `2 * n`
+            Object containing the elements of the rotation group.
+
+        Raises
+        ------
+        ValueError
+            If the group order is not a non-negative integer.
+        """
+
+        if not np.issubdtype(type(n), np.integer) or n < 1:
+            raise ValueError("n must be a positive integer")
+
+        g1 = self.cyclic(n, axis).as_rotvec()
+
+        thetas = np.linspace(0, np.pi, n, endpoint=False)
+        rv = np.pi * np.vstack([np.zeros(n), np.cos(thetas), np.sin(thetas)]).T
+        g2 = np.roll(rv, axis, axis=1)
+
+        return Rotation.from_rotvec(np.concatenate((g1, g2)))
+
+    @classmethod
+    def cyclic(self, n, axis=2):
+        """Get a cyclic rotation group.
+
+        Parameters
+        ----------
+        n : integer
+            The order of the group.
+        axis: integer
+            The cyclic rotation axis (default: 2)
+            (0: x-axis, 1: y-axis, 2: z-axis)
+
+        Returns
+        -------
+        rotation : `Rotation` instance of length `n`
+            Object containing the elements of the rotation group.
+
+        Raises
+        ------
+        ValueError
+            If the group order is not a non-negative integer.
+        """
+        if not np.issubdtype(type(n), np.integer) or n < 1:
+            raise ValueError("n must be a positive integer")
+
+        thetas = np.linspace(0, 2 * np.pi, n, endpoint=False)
+        rv = np.vstack([thetas, np.zeros(n), np.zeros(n)]).T
+        return Rotation.from_rotvec(np.roll(rv, axis, axis=1))
+
+    @classmethod
+    def fundamental(self, rotation, group):
```

Comment:
I thought that it can be made a proper method. Like `g = RotationGroup("O"), g.fundamental(r)`